lib/features/thread/presentation/widgets/email_tile_builder.dart

```diff
@@ -143,6 +146,17 @@ class EmailTileBuilder with BaseEmailItemTile {
     );
   }
 
+  Widget _buildCalenderIcon(PresentationEmail email) {
+    String mainString = email.emailHeader.toString();
+    String substring = "null";
```

Comment:
What is it?

Reply:
I could not get the header yet so just I use it for the test after getting the header I will put the condition there. I check if the emailheader contain the String  header:X-MEETING-UID: